src/components/Tabs/Tabs.js

```diff
@@ -18,109 +18,257 @@
-  height: 32px;
-  padding: 0 15px;
-  align-items: center;
+  height: ${props => props.isAndroid ? 53.5 : 32};
+  align-items: flex-start;
   justify-content: center;
-  background-color: ${props => props.active ? baseColors.electricBlue : 'transparent'};
+  background-color: ${props => props.active ? baseColors.white : 'transparent'};
   border-radius: 16px;
   flex-direction: row;
+  flex-grow: 1;
+  ${props => props.active
+    ? `
+    shadow-color: ${UIColors.tabShadowColor};
+    shadow-radius: 10px;
+    shadow-opacity: 1;
+    shadow-offset: 0px 7px;
+    `
+    : ''}
 `;
 
 const TabItemIcon = styled(Icon)`
   font-size: ${fontSizes.extraSmall};
-  margin-right: 5px;
-  color: ${props => props.active ? baseColors.white : baseColors.darkGray};
+  margin-right: 4px;
+  margin-top: 2px;
+  color: ${props => props.active ? baseColors.slateBlack : baseColors.electricBlue};
 `;
 
 const TabItemText = styled(BaseText)`
   font-size: ${fontSizes.extraSmall};
-  color: ${props => props.active ? baseColors.white : baseColors.darkGray};
+  font-weight: 500;
+  color: ${props => props.active ? baseColors.slateBlack : baseColors.electricBlue};
 `;
 
 const ActivityFeedHeader = styled.View`
   padding: 4px ${spacing.mediumLarge}px 0;
 `;
 
-const TabWrapper = styled.View`
-  padding: 12px 16px;
-  background: ${baseColors.white};
-  border-bottom-width: 1px;
-  border-color: ${UIColors.defaultDividerColor};
-  border-style: solid;
+const TextWrapper = styled.View`
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  height: 32px;
+  margin-top: ${Platform.select({
+    ios: '0',
+    android: '3px',
+  })}
+  ${props => props.extraPadding
+    ? `padding-left: 34px;
+      padding-right: 34px;`
+    : ''}
+`;
+
+const UnreadBadge = styled.View`
+  height: 24px;
+  width: 24px;
+  border-radius: 12px;
+  background-color: ${baseColors.electricBlue};
+  position: absolute;
+  top: ${Platform.select({
+    ios: '4px',
+    android: '4.5px',
+  })}
+  right: 2px;
+  align-items: center;
+  justify-content: center;
 `;
 
+const UnreadText = styled(BaseText)`
+  font-size: ${fontSizes.extraSmall};
+  font-weight: 500;
+  color: ${baseColors.white};
+`;
+
+const TabImage = styled(CachedImage)`
+  font-size: ${fontSizes.extraSmall};
+  font-weight: 500;
+  color: ${baseColors.white};
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+`;
+
+const tabBackground9Patch = require('assets/images/tab.png');
+
 export default class Tabs extends React.Component<Props, State> {
   state = {
-    activeTab: ALL,
+    activeTab: this.props.initialActiveTab || ALL,
+  };
+
+  renderIcon = (tab: Object) => {
+    const {
+      icon,
+      tabImageNormal,
+      tabImageActive,
+      id,
+    } = tab;
+    const { activeTab } = this.state;
+    const isActive = activeTab === id;
+    const tabImage = isActive ? tabImageActive : tabImageNormal;
+
+    if (icon || tabImageActive || tabImageNormal) {
+      if (icon) {
```

Comment:
Let's put this above the `if (icon || tabImageActive || tabImageNormal) {`

Reply:
Done :)